### DISCUSSION TITLE
Possibility of developing a "front-end" for nchat (with nchat running in the back-end)

### DISCUSSION THREAD

**Discussion by carlos-moreira** [d99kris/nchat#296] [Q&A]
Would it be possible to develop a web application where logged-in users of a specific system can connect their WhatsApp account and use it similarly to WhatsApp Web, but through a custom interface provided by the application? The back-end would be powered by nchat, while the front-end would interact with an API that communicates with nchat. This way, the application would act as a proxy for WhatsApp, allowing users to interact with it through the custom interface.

The goal is to develop an application with a support chat, where a support attendant can interact with clients using WhatsApp. However, my application needs to have access to all conversation logs, of course with the knowledge of the attendant.

**Comment by d99kris** [comment]
Hi @carlos-moreira - I am not sure. I mean "anything" is possible to implement given appropriate resources/etc, but the nchat project is not equipped with resources to support the features mentioned. Essentially it is outside the scope of nchat. Hope this clarifies, if not feel free to re-open.
